src/node/util.ts

```diff
@@ -227,24 +234,28 @@ function timestamp(): string {
   return `${d.getDate()} ${months[d.getMonth()]} ${t}`;
 }
 
-export function log(...args: any[]) {
+export function log(...args: unknown[]): void {
   console.log('%s - %s', timestamp(), format(...args));
 }
 
-export function parseArgs(..._: any[]): any {
+export function parseArgs(): unknown {
   // We currently have no plans to implement the util.parseArgs API.
   throw new Error('node:util parseArgs is not implemented');
 }
 
-export function transferableAbortController(..._: any[]): any {
+export function transferableAbortController(): void {
   throw new Error('node:util transferableAbortController is not implemented');
 }
 
-export function transferableAbortSignal(..._: any[]): any {
+export function transferableAbortSignal(): void {
   throw new Error('node:util transferableAbortSignal is not implemented');
 }
 
-export async function aborted(signal: AbortSignal, resource: object) {
+export async function aborted(
+  signal: unknown,
+  resource: unknown
+  // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
+): Promise<Event | void> {
```

Comment:
```suggestion
): Promise<Event | undefined>
```

Guessing that this is what eslint is complaining about